backend/src/api/announcement/presenters/announcement.presenter.ts

```diff
@@ -0,0 +1,117 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { AnnouncementStatus } from 'src/modules/announcement/enums/announcement-status.enum';
+import { IAnnouncementModel } from 'src/modules/announcement/models/announcement.model';
+import { OrganizationStructureType } from 'src/modules/organization/enums/organization-structure-type.enum';
+import { IOrganizationStructureModel } from 'src/modules/organization/models/organization-structure.model';
+
+class OrganizationStructureToAnnouncementPresenter {
+  constructor(structure: IOrganizationStructureModel) {
+    this.id = structure.id;
+    this.name = structure.name;
+    this.type = structure.type;
+    this.numberOfMembers = structure.members;
+  }
+
+  @Expose()
+  @ApiProperty({
+    description: 'The uuid of the Organization Structure',
+    example: '525dcdf9-4117-443e-a0c3-bf652cdc5c1b',
+  })
+  id: string;
+
+  @Expose()
+  @ApiProperty({
+    description: 'The name of the OrganizationStructure',
+    example: 'Financial',
+  })
+  name: string;
+
+  @Expose()
+  @ApiProperty({
+    description: 'The type of the structure (branch/department/role)',
+    enum: OrganizationStructureType,
+    examples: Object.values(OrganizationStructureType),
+  })
+  type: OrganizationStructureType;
```

Comment:
- probably not needed if there are only departments. 
- "OrganizationStructureListItemPresenter" will replace this if we add numberOfMembers which seems to be needed everywhere